aspnetcore/toc.yml

```
@@ -599,6 +599,8 @@ items:
                 uid: spa/react
               - name: JavaScript Services
                 uid: client-side/spa-services
+              - name: ASP.NET Core app with Vue >>
```

We'll be adding VS versions of the other SPA tutorials too, and what differentiates them from the ones in the toc now is their location in the VS docs.

```suggestion
              - name: Vue with Visual Studio >>
```